src/tersets.zig

```diff
@@ -68,5 +78,22 @@ pub fn decompress(
         .PoorMansCompressionMidrange, .PoorMansCompressionMean => {
             try pmc.decompress(compressed_values, decompressed_values);
         },
+        .SwingFilter => {
+            try swing_slide_filter.decompress(compressed_values, decompressed_values);
+        },
+    }
+}
+
+/// Auxiliary function to validate of the decompressed time series
```

Comment:
Unclear why a newline is added to the comment despite it being less than 100 characters. Maybe reformat the comment.

Reply:
Fixed.